backend/adapter_processor/adapter_processor.py

```diff
@@ -94,12 +93,6 @@ def test_adapter(adapter_id: str, adapter_metadata: dict[str, Any]) -> bool:
             test_result: bool = adapter_instance.test_connection()
             logger.info(f"{adapter_id} test result: {test_result}")
             return test_result
-        # HACK: Remove after error is explicitly handled in VertexAI adapter
-        except json.JSONDecodeError:
-            raise TestAdapterInputError(
-                "Credentials is not a valid service account JSON, "
-                "please provide a valid JSON."
-            )
```

Comment:
@chandrasekharan-zipstack why was this removed? And was this necessary for the change you were making?

Reply:
Missed to add this in the PR description - this was from an earlier change which has been made in the [SDK](https://github.com/Zipstack/unstract-sdk/blob/4acfa684369f2d647b927d8d3bf92ee142a58ec0/src/unstract/sdk/adapters/llm/vertex_ai/src/vertex_ai.py#L89-L92)

Reply:
Might have been a better idea to keep this as separate PR in that case.

Reply:
Will take note